RecoLocalCalo/EcalRecProducers/plugins/EcalUncalibRecHitWorkerMultiFit.cc

```diff
@@ -250,7 +250,6 @@ double EcalUncalibRecHitWorkerMultiFit::timeCorrection(
 }
 
 
-
 bool
 EcalUncalibRecHitWorkerMultiFit::run( const edm::Event & evt,
                 const EcalDigiCollection::const_iterator & itdg,
```

Comment:
First of all, even after fixing the variable name to `result` and adding a return value, the code you suggest to use
```c++
bool
EcalUncalibRecHitWorkerMultiFit::run( const edm::Event & evt,
                 const EcalDigiCollection::const_iterator & itdg, 
EcalUncalibratedRecHitCollection & result ){
EcalDigiCollection tempCollection(itdg);
run(evt, tempCollection, result);
return true;
```
does not even compile.

Then, if you truly believe that building a temporary `DataFrameContainer` for each ECAL cluster for event "there is ~0 extra cost" , I really do not know what to say.

If you insist, I can make a new PR with
```c++
bool
EcalUncalibRecHitWorkerMultiFit::run( const edm::Event & evt,
                const EcalDigiCollection::const_iterator & itdg,
                EcalUncalibratedRecHitCollection & result)
{
    throw edm::Exception(edm::errors::LogicError)
        << "Upon explicit request of Slava Krutelyov this function raises an unhandled exception rather than performing its function.\n"
        << "For more information please read the discussion at https://github.com/cms-sw/cmssw/pull/18600#pullrequestreview-37628902";

    return true;
}
```